src/Behavior.h

```diff
@@ -86,8 +86,8 @@ class Composite : public Behavior
 {
 public:
 	void addChild(Behavior* child) { m_Children.push_back(child); }
-	void removeChild(Behavior*);
-	void clearChildren();
+    void removeChild(Behavior*) { };
+    void clearChildren() { };
```

Comment:
je sais que ca viens du livre, mais on peut l'enlever pour l'instant

Reply:
Meh ça change pas grand chose de les laisser là, et si jamais on en a besoin pourra les réimplémenter. J'ai mieux une implémentation by the book que de commencer à changer des trucs pis d'être perdu éventuellement